catalog/spatial/geocoding/spatial-geocoding-admin-module/src/test/java/org/codice/ddf/spatial/admin/module/TestGeocodingMbean.java

```diff
@@ -54,6 +54,7 @@ public void setUp() {
     geoEntryIndexer = Mockito.mock(GeoEntryIndexer.class);
     geocoding.setGeoEntryIndexer(geoEntryIndexer);
     geocoding.setGeoEntryExtractor(geoEntryExtractor);
+    System.setProperty("ddf.home", System.getProperty("user.dir"));
```

Comment:
❓ Shouldn't we use a temporary directory here instead the user's home directory (`@TemporaryFolder` junit Rule)? That will ensure that nothing gets left behind after the tests are run.